index.js

```diff
@@ -113,6 +113,7 @@ class XiaomiRoborockVacuum {
     this.config.findMe = config.findMe || false;
     this.config.findMeWord = config.findMeWord || "where are you";
     this.config.delay = config.delay || false;
+    this.config.roomTimeout = config.roomTimeout == undefined ? 0 : config.roomTimeout;
```

Comment:
FYI, you can also write this as
```suggestion
    this.config.roomTimeout = config.roomTimeout ?? 0;
```

Reply:
Doesn't seem to work for me, maybe because I am sporting Node 12?